common/SC_ServerBootDelayWarning.cpp

```diff
@@ -0,0 +1,47 @@
+// Add a warning when Windows Defender delays scsynth boot by 60+ seconds
+// cf. github issue #4368
+#include "SC_ServerBootDelayWarning.h"
+
+#ifdef _WIN32
+#    include <iostream>
+#    include <thread>
+#    include <mutex>
+#    include <chrono>
+#    include <condition_variable>
+
+static bool g_isServerBooted = false;
+static std::mutex g_scsynthBootWarningMutex;
+static std::condition_variable g_scsynthBootWarningConditionVariable;
```

Comment:
anything using `scsynth` in its name should now use `server` instead

Reply:
oh yeah of course !